### PR TITLE
fix(warnings):CS1570

### PR DESCRIPTION
**Pull request by barustum**
CS1570: XML comment on 'construct' has badly formed XML

### CODE REVIEW

**Review by abhipsaMisra — APPROVED** [review]

File: iothub/device/src/ModuleClient.cs [hunk 1]
```diff
@@ -44,9 +44,7 @@ internal ModuleClient(InternalClient internalClient, ICertificateValidator certV
             }
 
             if (Logging.IsEnabled)
-            {
```
Comment (jamdavi) [inline iothub/device/src/ModuleClient.cs:47]: Did you intend to remove these? I haven't seen us choose one way or another in the code so not sure if you're confirming to a code style

File: iothub/service/src/DigitalTwin/Serialization/WritableProperty.cs [hunk 2]
```diff
@@ -13,6 +13,7 @@ namespace Microsoft.Azure.Devices.Serialization
     /// <para>
```
Comment (drwill-ms) [inline iothub/service/src/DigitalTwin/Serialization/WritableProperty.cs:13]: `<para>` tags not needed

**Review by drwill-ms — APPROVED** [review]

**Review by jamdavi — APPROVED** [review]

**Review by azabbasi — APPROVED** [review]

File: iothub/device/src/ModuleClient.cs (hunk 1 above)
Comment (barustum) [reply iothub/device/src/ModuleClient.cs:47]: Yes. I believe that's the agreed upon style for the Logging.IsEnabled block that we discussed. Looking at recent PRs, that's what we've been doing.

Comment (drwill-ms) [reply iothub/device/src/ModuleClient.cs:47]: We decided for the ubiquitous logging only, to reduce how many lines of code it takes.

**Pull request merged**